.circleci/config.yml

```diff
@@ -339,7 +339,6 @@ workflows:
             - setup
       - e2e-cli:
           name: e2e-cli
-          nodeversion: '14.15'
```

Comment:
With the package engine field using 14.15 as the minimum, I think we should consider keeping this node version for the E2E tests.